stdlib/smtplib.pyi

```diff
@@ -9,6 +9,8 @@ from types import TracebackType
 from typing import Any, Protocol, overload
 from typing_extensions import TypeAlias
 
+from _socket import _Address as _SourceAddress
```

Comment:
Oh, we should add `_socket` to this list, so isort recognises it as part of the stdlib:

https://github.com/python/typeshed/blob/9f0d0b9af7d4980431bbd62ca38497cf33308093/pyproject.toml#L15

Reply:
I can do that in a separate PR.

Reply:
#9098